hack/pr-infra-deployments.sh

```diff
@@ -23,10 +23,13 @@ set -o pipefail
 set -o nounset
 
 RELEASE_BUNDLE_TAG="git-$(git rev-parse --short HEAD)"
-RELEASE_BUNDLE_TAG="git-d47b70d"
```

Comment:
I don't think this was intentional, but lmk if it was.

Reply:
Oh no, thanks for fixing this, that was the only way to test locally...